### PR TITLE
Feature/typography

### PR DESCRIPTION
**Pull request by nuria-gs**
### ISSUE NUMBER
#51 Add in typography styles

### Labels
- IN REVIEW

### Describe the changes
Added simple-typography files in the sass folder and imported the files in styles.scss

### Screenshots (if appropriate)
"n/a" 

### Questions or comments (if any)
"n/a"

### CODE REVIEW

File: frontend/sass/style.scss [hunk 1]
```diff
@@ -0,0 +1,265 @@
+@import "_mixins.scss";
```
Comment (lili2311) [inline frontend/sass/style.scss]: please can you rename this file to be `_custom-typography.scss`

**Review by lili2311 — CHANGES_REQUESTED** [review]

**Comment by lili2311** [comment]
left a comment otherwise good to merge

**Comment by nuria-gs** [comment]
File renamed

**Comment by lili2311** [comment]
It is not present in the pull request, please can you push the new renamed file?

**Comment by nuria-gs** [comment]
Sorry, I didn't realize the renamed file was on the untracked list.

**Review by lili2311 — APPROVED** [review]

**Comment by lili2311** [comment]
merging ty!

**Pull request merged**